bindata/network/multus/multus.yaml

```diff
@@ -126,7 +126,8 @@ data:
         "perNodeCertificate": {
           "enabled": true,
           "bootstrapKubeconfig": "/hostroot/var/lib/kubelet/kubeconfig",
-          "certDir": "/run/multus/certs"
+          "certDir": "/etc/cni/multus/certs",
+          "certDuration": "24h"
```

Comment:
NIT: We could probably make this a variable if in the future we want to enable customizing that value